website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -220,10 +217,10 @@ To peer Kubernetes clusters running Consul, you need to create a peering token a
     $ kubectl apply --filename frontend-service.yml
     ```
 
-1. Run the following command and check the output to confirm that you peered your clusters successfully.
+1. Run the following command in `frontend-service` and check the output to confirm that you peered your clusters successfully.
 
     ```shell-session
-    $ curl localhost:1234
+    $ kubectl exec -it $(kubectl get pod -l app=frontend -o name) -- curl localhost:1234
```

Comment:
With the `consul.hashicorp.com/connect-service-upstreams` annotation removed above, I think this needs to be updated to use the DNS name for the imported service instead of the localhost port?